### PR TITLE
13507 fix civic address validation errors

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity#13507

*Description of changes:*

- Fix validations. See #13507
- Fix Form validation so registration can be submitted

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue [hunk 1]
```diff
@@ -76,14 +76,14 @@
               <h3>Civic Address</h3>
             </v-col>
             <v-col cols="9" class="pt-1">
-              <p v-if="getMhrRegistrationLocation.address.street &&
-                       getMhrRegistrationLocation.address.city">
+              <p v-if="getMhrRegistrationLocation.address.street ||
```
Comment (dimak1) [inline ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue:79]: Not sure about this logic here. With the `||` operator, if `address.streetAdditional` is defined we still display `address.street` that could be undefined. Could cause some errors.

File: ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue [hunk 2]
```diff
@@ -130,11 +132,19 @@ export default defineComponent({
...
-    const isValidCivicAddress = false
+    const localState = reactive({
+      isValidCivicAddress: false,
+      addressLocal
+    })
+
+    const validateForm = (): void => {
+      if (props.validate) {
+        // @ts-ignore - function exists
+        if (context.$refs.street.valid && context.$refs.city.valid) localState.isValidCivicAddress = true
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue]: Could this be simplified to `localState.isValidCivicAddress = context.$refs.street.valid && context.$refs.city.valid` ?

File: ppr-ui/src/utils/feature-flags.ts [hunk 3]
```diff
@@ -9,8 +9,8 @@ export const defaultFlagSet: LDFlagSet = {
   'bcregistry-ui-mhr-enabled': false,
   'search-registration-number': false,
   'search-serial-number': false,
-  'mhr-ui-enabled': false,
-  'mhr-registration-enabled': false
+  'mhr-ui-enabled': true,
+  'mhr-registration-enabled': true
```
Comment (cameron-eyds) [inline ppr-ui/src/utils/feature-flags.ts]: Might want to revert the defaults here

**Review by cameron-eyds — COMMENTED** [review]
Looks good so far, just a few comments to tackle!

**Review by doug-lovett — APPROVED** [review]
No additional comments.

File: ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue (hunk 1 above)
Comment (ChasBoucher) [reply ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue:79]: I did some testing and there is no error returned if null,'', undefined...   I did find a potential issue in the validation, so fixed that.

File: ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue (hunk 2 above)
Comment (ChasBoucher) [reply ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue]: In changing to your suggestion. I found another issue, regarding the context, so fixed that.  I did some thorough testing. I will be testing more on my next ticket, but validation seems to catch all exceptions..

File: ppr-ui/src/utils/feature-flags.ts (hunk 3 above)
Comment (ChasBoucher) [reply ppr-ui/src/utils/feature-flags.ts]: Changed values to false..

**Review by cameron-eyds — APPROVED** [review]
Thanks for the update, looks good.

File: ppr-ui/src/utils/feature-flags.ts (hunk 3 above)
Comment (cameron-eyds) [reply ppr-ui/src/utils/feature-flags.ts]: Did you commit this change?

Comment (ChasBoucher) [reply ppr-ui/src/utils/feature-flags.ts]: I just did... Did one last test after changing to false.. Missed it in checkin.

Comment (cameron-eyds) [reply ppr-ui/src/utils/feature-flags.ts]: Sounds good, thanks!

**Review by dimak1 — APPROVED** [review]

**Pull request merged**